### ISSUE TITLE
vue good tables don't work on IE10

### ISSUE THREAD

**Issue by leonardochen30** [xaksis/vue-good-table#153]
On IE10 vue good tables is not working, send syntax error, vue-good-demos also does not run on IE10.

**Comment by leonardochen30** [comment]
i resolve the issue adding this **resolve('node_modules/vue-good-table/src')** to the babel-loader include section.

**Comment by xaksis** [comment]
hey @leonardochen30 can you explain how you were including the plugin before? technically you shouldn't need to be referring to the source files. Plugin exposes the transpiled es5 version of the component.

**Comment by leonardochen30** [comment]
hi @xaksis, I followed the installation instructions with the command npm install --save vue-good-table, in my package.json was added in the dependencies section, "vue-good-table": "^ 1.18.2".
Then in my main file main.js I import it in the following way import VueGoodTable from 'vue-good-table'; and then Vue.use (VueGoodTable);
From what I saw in IE10, it did not work because it included the reserved word const instead of var.
My project is created with the following command: npm install --global vue-cli and vue init webpack my-project, npm install.
I also download the example project vue-good-example, I gave npm install and it did not work in IE10 for the same error.
I attach my configuration files.
[SACPVue.zip](https://github.com/xaksis/vue-good-table/files/1622831/SACPVue.zip)
